src/System.Management/src/System/Management/InteropClasses/WMIInterop.cs

```diff
@@ -0,0 +1,2351 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Runtime.Serialization;
+using System.Threading;
+using System.Security.Permissions;
+using System.Runtime.Versioning;
+
+namespace WbemClient_v1 {}
+namespace WbemUtilities_v1 {}
```

Comment:
Can these be removed? What's the purpose of these empty namespaces?

Reply:
Thanks Justin, likely yes. What is their purpose? No idea... I've been looking more at build issues and want to get it in before starting really changing the sources but this one should be fine to remove now. Please, keep reporting what you see.